### PR TITLE
Reset robot pose by z key

### LINKED ISSUE #43: Reset robot pose by z key.
urdf supports moving the robot by http request.
But it is not possible to reset by key short cut.
I want to support it by `z` key.

As a work around, it can be done by `l` key (reload urdf).

### PR DESCRIPTION
**Pull request by taiki-e**
Closes #43

https://user-images.githubusercontent.com/43724913/114505401-49e35100-9c6b-11eb-9242-0bab88270b9c.mov

### CODE REVIEW

File: src/bin/urdf-viz.rs [hunk 1]
```diff
@@ -355,10 +355,14 @@ impl UrdfViewerApp {
                 }
             }
             Key::Z => {
+                let mut origin = self.robot.origin();
+                origin.translation = na::Translation3::new(0.0, 0.0, 0.0);
+                origin.rotation = na::UnitQuaternion::from_euler_angles(0.0, 0.0, 0.0);
+                self.robot.set_origin(origin);
```
Comment (OTL) [inline src/bin/urdf-viz.rs:361]: I think you can do like below (I've not checked anything).

```suggestion
              self.robot.set_origin(na::Isometry::identity());
```

**Review by OTL — COMMENTED** [review]
And could you update the how to use in the window?

```text
z: reset joint positions and origin
```

File: src/bin/urdf-viz.rs (hunk 1 above)
Comment (taiki-e) [reply src/bin/urdf-viz.rs]: Thanks. That's works well.

**Comment by taiki-e** [comment]
Updated usage and applied a suggestion.

s

**Review by OTL — APPROVED** [review]
Thank you great!!

**Pull request merged**